docs/data/material/components/tabs/NavTabs.tsx

```diff
@@ -24,7 +42,9 @@ export default function NavTabs() {
   const [value, setValue] = React.useState(0);
 
   const handleChange = (event: React.SyntheticEvent, newValue: number) => {
-    setValue(newValue);
+    if (event.type !== 'click' || samePageLinkNavigation(event as any)) {
```

Comment:
Regarding `event.type !== 'click'` - does this mean we change tabs only on `mouseup` or `mousedown` mouse events?

Reply:
tabs value change can be triggered with tabs, space and enter.